core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java

```diff
@@ -177,12 +177,9 @@ public void display(PrintWriter out, TextField field, FieldTemplate template,
 
   private void displayContent(PrintWriter out, String code, PagesContext pageContext) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "pageContext". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)